eng/common/mcp/azure-sdk-mcp.ps1

```diff
@@ -0,0 +1,60 @@
+#!/bin/env pwsh
+
+param(
+    [string]$FileName = 'azsdk',
+    [string]$Package = 'azsdk',
+    [string]$Version, # Default to latest
```

Comment:
Should we hard-code the pinned version here?

Reply:
We can. Wasn't sure if we wanted to take the eng/common sync approach to versioning, or just doing a publish to the releases page.

I may switch it to use the text file approach we do with proxy for consistency.

Reply:
I'm fine with reading it from a file if that is easier but we do have other cases where we hardcode the version in a template or script. As long as it is only in one place it probably doesn't matter which.

Reply:
> I'm fine with reading it from a file if that is easier but we do have other cases where we hardcode the version in a template or script. As long as it is only in one place it probably doesn't matter which.

Having it read from a file allows other common tooling in the repos an easy place to figure out what version they _should_ be on if they integrate as well.